chat_filter.user.js

```diff
@@ -814,32 +814,17 @@ add_initializer(function(){
 
     add_custom_css([
         ".chat-room { z-index: inherit !important; }",
+
         ".chat-settings { z-index: 100 !important; }",
+        ".chat-settings { max-height: 500px; }",
```

Comment:
I thought it was more readable this way. I don't like putting more than one CSS rule per line.